src/core/XHP.php

```diff
@@ -21,7 +21,7 @@ public function __construct(
   }
   abstract public function appendChild(mixed $child): this;
   abstract public function prependChild(mixed $child): this;
-  abstract public function replaceChildren(...): this;
+  abstract public function replaceChildren(XHPChild ... $children): this;
```

Comment:
This is a type error in my version of hhvm.
`Typing[4223] Cannot use ... without a type hint and parameter name in strict mode.`
I don't know how long xhp-lib@v3 has been unusable because of this. 😕